packages/naming.presets/test/cell.test.js

```diff
@@ -0,0 +1,304 @@
+'use strict';
+
+const expect = require('chai').expect;
+
+const BemCell = require('@bem/sdk.cell');
+const BemEntityName = require('@bem/sdk.entity-name');
+
+const createStringify = require('@bem/sdk.naming.cell.stringify');
+
+const presets = require('..');
+
+const createPreset = (name, fsConv, conv) => {
+    const res = Object.assign({}, presets[name], conv);
+    res.fs = Object.assign({}, res.fs, fsConv);
+    return res;
+};
+
+const n = require('path').normalize;
```

Comment:
why do we need to normalize paths ?
it's `n('common.blocks/a/_mn/a_mn_mv.js')` already normalized

Reply:
Because of WINdowns.